src/games/strategy/debug/DebugUtils.java

```diff
@@ -76,14 +77,16 @@ public static String getMemory() {
     }
     return buf.toString();
   }
-
-  @SuppressWarnings({"unchecked", "rawtypes"})
   public static String getProperties() {
     final StringBuilder buf = new StringBuilder("SYSTEM PROPERTIES\n");
     final Properties props = System.getProperties();
-    final java.util.List keys = new ArrayList(props.keySet());
+    final List<Object> keysObject = new ArrayList<>(props.keySet());
```

Comment:
Still a some unnecessary casting happening, can condense this 14 line method down to:
```
    final StringBuilder buf = new StringBuilder("SYSTEM PROPERTIES: \n");
    final Properties props = System.getProperties();

    props.keySet().forEach(keyObject -> {
      final String property = keyObject.toString();
      final String value = props.getProperty(property);
      buf.append(property).append(" ").append(value).append("\n");
    });

    return buf.toString();
```